### PR TITLE
Feature/front end fixing

### PR DESCRIPTION
**Pull request by annielin28815**
1. modify style of homepage 
2. modify style of forms and buttons
2. fix input of searchbar

### CODE REVIEW

**Review by xx4673 — APPROVED** [review]
checked

File: app/views/ticket_types/choose_ticket.html.erb [hunk 1]
```diff
@@ -26,17 +26,6 @@
...
-            <div>
-              <span>開售日期：</span>
-                <%= ticket.sell_start.strftime("%Y/%m/%d") %>
-              <span>停售日期：</span>
-                <%= ticket.sell_deadline.strftime("%Y/%m/%d") %>
-              <span>狀態：</span>
-                <%= content_tag :span, '未開售', class:"p-2 bg-blue-400 textg-blue-800 rounded-lg" if ticket.pending? %>
-                <%= content_tag :span, '銷售中', class:"p-2 bg-teal-400 text-teal-800 rounded-lg" if ticket.on_sale? %>
-                <%= content_tag :span, '售完', class: "p-2 bg-yellow-400 text-yellow-800 rounded-lg" if ticket.fully_booked? %>
-                <%= content_tag :span, '停止販售', class:"p-2 bg-red-400 text-red-800 rounded-lg" if ticket.sale_closed? %>
```
Comment (samliuforwork) [inline app/views/ticket_types/choose_ticket.html.erb:38]: @xx4673 這邊狀態之後不放了嗎

**Review by samliuforwork — APPROVED** [review]
Checked

**Review by kihifung — APPROVED** [review]
checked

**Pull request merged**